src/board_controller/build.cmake

```diff
@@ -188,6 +188,8 @@ if (MSVC)
 endif (MSVC)
 if (UNIX AND NOT ANDROID)
     add_custom_command (TARGET ${BOARD_CONTROLLER_NAME} POST_BUILD
+        COMMAND "${CMAKE_COMMAND}" -E copy_if_different "${CMAKE_CURRENT_SOURCE_DIR}/compiled/${BOARD_CONTROLLER_COMPILED_NAME}" "${CMAKE_CURRENT_SOURCE_DIR}/swift_package/BrainFlow/lib/${BOARD_CONTROLLER_COMPILED_NAME}"
+        COMMAND "${CMAKE_COMMAND}" -E copy_if_different "${CMAKE_CURRENT_SOURCE_DIR}/compiled/${BOARD_CONTROLLER_COMPILED_NAME}" "${CMAKE_CURRENT_SOURCE_DIR}/swift_package/BrainFlowCI/Frameworks/${BOARD_CONTROLLER_COMPILED_NAME}"
```

Comment:
as discussed in slack, from the c++ level we should not copy libs to tests and examples for bindings

Reply:
My mistake. I will correct those files tonight.

Reply:
There are a few more unresolved comments from orevious pass, lets go through all of them. 

But overall - great job! Its just minor stuff

Reply:
updated